### ISSUE TITLE
HEXDIG wrong

### ISSUE THREAD

**Issue by mxmlnkn** [michaelrsweet/abnf#7]
For the simplest rule `h = HEXDIG`, the online converter gives me:
```regex
^([0-9]|%[Xx]41\-46|%[Xx]61\-66)$
```
which matches things like:
```
9
%x41-46
%X41-46
```
and should relatively surely be instead:
```regex
^[0-9A-Fa-f]$
```

**Comment by michaelrsweet** [comment]
Will investigate - it *should* just substitute the characters (and *was* working...)

**Comment by michaelrsweet** [comment]
Confirmed this is the case. Investigating this further...

**Comment by michaelrsweet** [comment]
[master 5ab1955] Fix HEXDIG core rule (Issue #7)

After staring at it and adding some debug messages I realized that the HEXDIG definition had double quotes around the character constants... Fixed in Git master, will push to abnf.msweet.org once I've made the rest of the fixes.
